### PR TITLE
Write bylaws

### PR DESCRIPTION
**Pull request by ArchdukeTim**

### CODE REVIEW

File: bylaws.tex [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+In the event that the number of elected Directors falls to less than four, through resignations or any other cause, the Board of Directors shall take no action other than the election of Directors until the number of elected Directors is at least four.
+
+\section{Staggered Terms}
+
+The elected Directors shall be divided into three groups about equal in number with the Directors comprising each group having terms of office that expire in a different year than of Directors in the other groups, so that the terms of office of about one-third of elected Directors expire each year.
+
+\section{Regular Meetings}
+
+A regular annual meeting of the Board of Directors shall be held in September of each year, at such time and place as shall be designated by the President or Vice President in the notice of the meeting for the purpose of electing Directors and officers and for the transaction of such other business as may come before the meeting. The Board of Directors may provide by resolution for the holding of additional regular meetings.
```
Comment (ArchdukeTim) [inline bylaws.tex]: I think we should change this to be August, so that the meeting can happen prior to the academic year starting

File: bylaws.tex [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+\section{Quorum}
+
+Four Directors present in person shall constitute a quorum for the transaction of business at any meeting of the Board of Directors, unless the presence of a greater number is required by law, in which case the quorum shall be such number as the law requires.
```
Comment (ArchdukeTim) [inline bylaws.tex]: Does there need to be a clause if there aren't 4 directors on the board?

File: bylaws.tex [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+
+\section{Procedure}
+
+The Chairman of the Board of Directors, or in his absence, the President, shall preside at meetings of the Board of Directors. The Board of Directors may adopt its own rules of procedure which shall not be inconsistent with these Bylaws.
+
+\chapter{Officers}
+
+\section{Officers}
+
+The officers of the Corporation shall be a President, a Vice President, a Secretary, and a Treasurer. The Corporation may, at the direction of the Board of Directors, have additional officers including, without limitation, other vice presidents, one or more assistant secretaries, and one or more assistant treasurers. One person may hold two or more offices except one person may not hold both the offices of President and Secretary.
```
Comment (ArchdukeTim) [inline bylaws.tex]: One person shouldn't hold president and treasurer as well I think

File: bylaws.tex (hunk 2 above)
Comment (rrlapointe) [reply bylaws.tex]: There will always be at least three Directors because of the three ex officio Directors. If there are less than four Directors on the Board that would mean there are zero elected Directors. I assume that won't happen, especially because the Board isn't allowed to do anything but elect Directors once the number of elected Directors gets less than four. So it reaching zero would be pretty difficult I imagine.

File: bylaws.tex (hunk 3 above)
Comment (rrlapointe) [reply bylaws.tex]: ok

File: bylaws.tex (hunk 1 above)
Comment (rrlapointe) [reply bylaws.tex]: ok

File: bylaws.tex (hunk 2 above)
Comment (ArchdukeTim) [reply bylaws.tex]: As long as there isn't conflicting bylaws that require quorum for new directors this is fine

Comment (rrlapointe) [reply bylaws.tex]: A quorum is required in order to elect new directors. But the only way a quorum of four would be unobtainable would be in the event that there are zero elected directors left, leaving only the three ex officio directors (who can't resign). Given that the bylaws also say the following, my reasoning is that it would be highly improbable to end up in the situation where you can't obtain quorum.

> In the event that the number of elected Directors falls to less than four, through resignations or any other cause, the Board of Directors shall take no action other than the election of Directors until the number of elected Directors is at least four.

In other words if the total number of directors falls to less than seven the board must elect more before it does anything else. In order to hit the scenario where you can't obtain quorum, all of the elected directors would have to resign at once. Because you seem to still be concerned about that scenario, I'll make an adjustment so that if the number of directors is less than four then quorum will be three. But that should never happen.

Comment (rrlapointe) [reply bylaws.tex]: The difference between "directors", "elected directors", and "ex officio directors" is significant.

Comment (rrlapointe) [reply bylaws.tex]: Just fixed it in 98d193958059c06ce4c54c4e612c55e1cbc8d259 so that even if all of the elected directors resign, the three ex officio directors will still be able to elect more directors.
